recap af experiments (#70)
thesis20/report: sections/experiments/results.tex

\subsection{Results of experiments}\label{subsec:resultsofexperiment}

> In the experiments we ran a number of methods on four different datasets that contain either contextual information or side-information.

Reviewer: ```suggestion
In the experiments, we ran a number of methods on four different datasets that contain either contextual information or side-information.
```

> The methods include two context-aware methods, some state-of-the-art graph-based methods and some simple baselines.

Reviewer: ```suggestion
The methods included two context-aware methods, some state-of-the-art graph-based methods, and some simple baselines.
```

> These methods are evaluated on a number of different metrics that show the performance of the methods on the different datasets.

Reviewer: ```suggestion
These methods were evaluated on a number of different metrics that showed their performance on the different datasets.
```
